ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusSticker.qml

```diff
@@ -15,11 +14,9 @@ Loader {
         height: 140
     }
 
-    signal loaded()
+    signal stickerLoaded()
```

Comment:
To prevent shadowing a builtin signal, `Loader` already has this signal: https://doc.qt.io/qt-5/qml-qtquick-loader.html#loaded-signal